packages/playground/public/info/jitsi.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Jitsi
+---
+
+Jitsi Meet is a set of Open Source projects which empower users to use and deploy video conferencing platforms with state-of-the-art video quality and features. For more details, check [Jitsi documentation](https://jitsi.github.io/handbook/docs/intro/).
```

Comment:
The link should redirect to app manual link. The link should work once info_grid PR is merged.
```suggestion
Jitsi Meet is a set of Open Source projects which empower users to use and deploy video conferencing platforms with state-of-the-art video quality and features. For more details, check [Jitsi documentation](https://www.manual.grid.tf/documentation/dashboard/solutions/jitsi.html).
```

Reply:
done, applied suggestion